terraform/deployments/chat/redis.tf

```diff
@@ -0,0 +1,46 @@
+locals {
+  chat_redis_name     = "chat-redis"
+  elasticache_subnets = data.terraform_remote_state.infra_networking.outputs.private_subnet_elasticache_ids
+}
+
+resource "aws_elasticache_subnet_group" "chat_redis_cluster" {
+  name       = local.chat_redis_name
+  subnet_ids = local.elasticache_subnets
+}
+
+resource "aws_security_group" "chat_redis_cluster" {
+  name        = local.chat_redis_name
+  vpc_id      = data.tfe_outputs.vpc.nonsensitive_values.id
+  description = "GOV.UK Chat Redis cluster"
+  tags = {
+    Name   = local.chat_redis_name
+    System = "Sidekiq"
```

Comment:
We are expecting that we'll use Redis for other things than Sidekiq, such as tracking rate limits.

Reply:
Right, I guess we can remove this tag for now, I'm not sure it's used for anything beyond being descriptive.

Reply:
removed